spec/features/fill_in_whole_form_spec.rb

```diff
@@ -176,5 +201,76 @@
     expect(employer_contract_claim_table.claim_information_row.claim_information_answer).to have_text "lorem ipsum info"
```

Comment:
I cant see anywhere that you are testing the reference number or the submitted date ?  I would have expected these to be tested against the mock values - and it would have failed because the hard coded values are still in the page :-)

Reply:
Implemented over 4f43bf8 and 216c6ec.

This is a really good point, I was terminating my test prematurely without testing core functionality.